st2common/st2common/models/db/execution.py

```diff
@@ -39,16 +38,7 @@
 class ActionExecutionDB(stormbase.StormFoundationDB):
     RESOURCE_TYPE = ResourceType.EXECUTION
     UID_FIELDS = ["id"]
-
```

Comment:
liveaction was changed from an embedded dict to a string.  all other changes here are cosmetic